### PR TITLE
Refine INVALID_SIGNATURE codes for SC precompile signing requirements

### PR DESCRIPTION
**Pull request by tdermendzhievv**
**Description**:
with this PR are introduced new error codes for not valid full prefix signature for specific precompile. 
This PR is connected with https://github.com/hashgraph/hedera-protobufs/pull/213

**Related issue(s)**: #3076

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/impl/TokenCreatePrecompile.java [hunk 1]
```diff
@@ -209,8 +209,8 @@ public void run(final MessageFrame frame) {
 		final var treasuryId = Id.fromGrpcAccount(tokenCreateOp.getTreasury());
 		final var treasuryHasSigned = KeyActivationUtils.validateKey(
 				frame, treasuryId.asEvmAddress(), sigsVerifier::hasActiveKey, ledgers, updater.aliases());
-		validateTrue(treasuryHasSigned, INVALID_SIGNATURE);
-		tokenCreateOp.getAdminKey().ifPresent(key -> validateTrue(validateAdminKey(frame, key), INVALID_SIGNATURE));
+		validateTrue(treasuryHasSigned, INVALID_FULL_PREFIX_SIGNATURE_FOR_PRECOMPILE, "Invalid full prefix for token create precompile!");
```
Comment (Nana-EC) [inline hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/impl/TokenCreatePrecompile.java:212]: nit: would be cleaner to make all the "Invalid full prefix ...." static properties in these classes

**Review by Nana-EC — DISMISSED** [review]
LG
Could improve with static fields for the error messages in the Precompile classes

**Review by shemnon — DISMISSED** [review]

**Comment by shemnon** [comment]
Good change, but it looks like we need to wait for the protobuf changes to be integrated into the codebase.

File: hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/impl/TokenCreatePrecompile.java (hunk 1 above)
Comment (IvanKavaldzhiev) [reply hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/impl/TokenCreatePrecompile.java:212]: We can have a common property with value "Invalid full prefix for %s precompile!" and different static values for each precompile (`"mint"`, `"burn"`, `"associate"`, etc.) and then do `String.format(INVALID_FULL_PREFIX, MINT)`

**Review by stoqnkpL — APPROVED** [review]
LGTM!

**Pull request merged**